bodhi/server/ffmarkdown.py

```diff
@@ -74,7 +74,7 @@ def handleMatch(self, m):
             return el
 
     MENTION_RE = r'(?<!\S)(@\w+)'
-    BUGZILLA_RE = r'([\S]+)(#[0-9]{5,})'
+    BUGZILLA_RE = r'([a-zA-Z]+)(#[0-9]{5,})'
```

Comment:
I recommend adding a test case or two using the strings from the bug report to demonstrate the fix